kmk/modules/pimoroni_trackball.py

```diff
@@ -1,13 +1,13 @@
-'''
+"""
```

Comment:
Single quotes are the prefered string denotation. Please revert all double quotes to single quotes where applicable.